Documentation/ViewDesigner/ConversionOptions.md

```diff
@@ -0,0 +1,26 @@
+# <strong>Conversion Options</strong>
+
+View Designer has a fixed static style of representation, while Workbooks enables freedom to include and modify how the data is represented, below depicts a few examples of how one might transform the views within Workbooks.
+
+[View Designer - Vertical](https://go.microsoft.com/fwlink/?linkid=874159&resourceId=%2Fsubscriptions%2F5c038d14-3833-463f-a492-de956f63f12a%2Fresourcegroups%2Faul-rg%2Fproviders%2Fmicrosoft.operationalinsights%2Fworkspaces%2Faul-test&featureName=Workbooks&itemId=%2Fsubscriptions%2F5c038d14-3833-463f-a492-de956f63f12a%2Fresourcegroups%2Faul-rg%2Fproviders%2Fmicrosoft.insights%2Fworkbooks%2F5a0ba062-7246-4907-b03f-eed8b55bf1f7&workbookTemplateName=View%20Designer%20Vertical&func=NavigateToPortalFeature&type=workbook)

```

Comment:
Need to figure out what to do with these links - for basically everyone will fail w/ some message of 'workbook not found.
Some options:
- Find a workspace that 'Everyone' has access to, use that (may be security and other considerations)
- Check-in them in here w/ steps to re-create
- Add them to Workspace Gallery